features/seeds/fixtures/lower_publicBody_complete_active_registration.json

```diff
@@ -0,0 +1,70 @@
+{
+    "location" : "england",
+    "businessType" : "soleTrader",
```

Comment:
Why it say to be a publicBody registration but the business type here is `soleTrader`? This will potentially cause confusion and broken data :)

Reply:
Well spotted! 🧐 Now fixed!